api/src/database/migrations/20211004A-add-project-language.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
+import { getHelpers } from '../helpers';
+
+export async function up(knex: Knex): Promise<void> {
+	const helper = getHelpers(knex).schema;
+
+	await knex.schema.alterTable('directus_settings', (table) => {
+		table.string('project_language').notNullable().defaultTo('en-US');
```

Comment:
Lets call the field `default_language` instead of `project_language` to avoid confusion with the languages configured by the individual users